<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_number</class>
 <widget class="QDialog" name="Set_number">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>510</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;-X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;지움&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;확인&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_dot">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;취소&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_unit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_number_zone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>337</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dot</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_dot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_14</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num_back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_11</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>click_num_clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_12</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>set_num_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_16</sender>
   <signal>clicked()</signal>
   <receiver>Set_number</receiver>
   <slot>set_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_num()</slot>
  <slot>click_dot()</slot>
  <slot>click_num_back()</slot>
  <slot>click_num_clear()</slot>
  <slot>set_num_save()</slot>
  <slot>set_cancel()</slot>
 </slots>
</ui>
